group_project/assignment_5.py

```diff
@@ -1,6 +1,20 @@
-def positions(a_string, first_word, second_word, third_word):
-    pass
-
+def positions(a_string, first_word = None, second_word = None, third_word = None):
+    if str(a_string.find(first_word)) == "-1":
```

Comment:
Why are you turning it into a string? You could have just done `a_string.find(first_word) == -1`.

Reply:
why do you have == -1  ?

Reply:
It's a way checking if it isn't there. If it's -1 then it doesn't have position/isn't found and can return "-". It can also be done with < 0.